content/en/blog/_posts/2021-00-00-kpng-specialized-proxiers.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: blog
+title: "Use KPNG to write specialized kube-proxiers"
+date: 2021-09-00
+slug: use-kpng-to-write-specialized-kube-proxiers
+---
+
+Authors: Lars Ekman, Ericsson
+
+The post describes how you can create a specialized proxier using
+Kubernetes Proxy NG (kpng) without interfering with the existing
+kube-proxy. The kpng project aims at renewing the Kubernetes internal
+load-balancer, the "kube-proxy". An important feature of kpng is that
```

Comment:
"internal load-balancer" can lead to confusions with the Service with Type LoadBalancer,  it is more like "the default Kubernetes Services implementation" or something like that.

Reply:
Good point. I'll update that